catroid/src/main/java/org/catrobat/catroid/ui/dialogs/RestrictedLoginDialog.java

```diff
@@ -89,7 +88,7 @@ public void onClick(View view) {
 								.apply();
 
 						if (!Utils.isUserLoggedIn(getActivity()) || !Utils.isCreateAtSchoolUser(getActivity())) {
-							ProjectManager.getInstance().showLogInDialog(getActivity(), false);
+							Utils.showPrivacyPolicyDialogWithLoginDialogAction(getActivity());
```

Comment:
is this also shown when the user has already accepted the privacy policy?

Reply:
No, then the LoginDialog is shown. The method name was a bit confusing - I've changed that.